pygments/lexers/_lilypond_builtins.py

```diff
@@ -0,0 +1,5148 @@
+# -*- coding: utf-8 -*-
+"""
+    pygments.lexers._lilypond_builtins
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+    LilyPond builtins.
+
+    :copyright: Copyright 2021-2021 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+# Contents generated by the script lilypond-builtins-generator.ly
+# found in the external/ directory of the source tree.
+
+keywords = [
+  "accepts",
+  "addlyrics",
+  "alias",
+  "book",
+  "bookpart",
+  "chordmode",
+  "chords",
+  "consists",
+  "context",
+  "defaultchild",
+  "denies",
+  "description",
+  "drummode",
+  "drums",
+  "etc",
+  "figuremode",
+  "figures",
+  "header",
+  "include",
+  "inherit-acceptability",
+  "language",
+  "layout",
+  "lyricmode",
+  "lyricsto",
+  "markup",
+  "markuplist",
+  "midi",
+  "name",
+  "new",
+  "notemode",
+  "once",
+  "override",
+  "paper",
+  "remove",
+  "revert",
+  "score",
+  "set",
+  "temporary",
+  "tweak",
+  "type",
+  "undo",
+  "unset",
+  "version",
+  "with",
+]
+
+clefs = [
+  "C",
+  "F",
+  "G",
+  "GG",
+  "alto",
+  "altovarC",
+  "baritone",
+  "baritonevarC",
+  "baritonevarF",
+  "bass",
+  "french",
+  "hufnagel-do-fa",
+  "kievan-do",
+  "mensural-f",
+  "mensural-g",
+  "mezzosoprano",
+  "moderntab",
+  "percussion",
+  "petrucci-f",
+  "petrucci-g",
+  "soprano",
+  "subbass",
+  "tab",
+  "tenor",
+  "tenorG",
+  "tenorvarC",
+  "treble",
+  "varC",
+  "varbaritone",
+  "varpercussion",
+  "violin",
+]
+
+scales = [
+  "aeolian",
+  "dorian",
+  "ionian",
+  "locrian",
+  "lydian",
+  "major",
+  "minor",
+  "mixolydian",
+  "phrygian",
+]
+
+repeat_types = [
+  "percent",
+  "unfold",
+  "volta",
+]
+
+pitches = [
+  "R",
+  "a",
+  "a",
+  "a",
+  "a",
+  "a",
+  "a",
+  "a",
+  "a-flat",
+  "a-flatflat",
+  "a-natural",
+  "a-sharp",
+  "a-sharpsharp",
+  "ab",
+  "acousticbassdrum",
+  "acousticsnare",
+  "ad",
+  "adb",
+  "add",
+  "aeh",
```

Comment:
Is it intentional that there are so many duplicates?

Reply:
They come from different languages. I've used `uniq-list` to avoid this.